GSA/openacr-editor
# Chapter link works

Changes:
* Created chapter component and updated navigation to use it.
* Installed typescript and rollup-plugin-typescript packages to support loading typescript from OpenACR package.
* Updated rollup and installed @rollup/plugin-commonjs packages to support the typescript that has mixing JavaScript syntax.
* Added OpenACR validation.

QA the navigation
1. After deployment ('Deploy' action) is successful, open site https://gsa.github.io/openacr-editor/.
2. Click the different A/Hardware/... navigation elements.
3. Confirm the pages do load (no error or odd behavior) and open up a page showing the full title of the OpenACR chapter.
4. Test the previous and next navigation and confirm those links also work.

QA the validation
1. After deployment ('Deploy' action) is successful, open site https://gsa.github.io/openacr-editor/.
2. Click 'Start new report'
3. Type in some values and random text in the 'License' field.
4. Click the 'View report' or 'Report' tab.
5. Confirm you see an 'Invalid' message about the license.
6. Go back to 'About' and set the license to the value 'GPL-2.0-or-later'.
7. Click the 'View report' or 'Report' tab.
8. Confirm you see a 'Valid' message.
